plugins/discourse-local-dates/spec/system/local_dates_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+describe "Local dates", type: :system, js: true do
+  fab!(:topic) { Fabricate(:topic) }
```

Comment:
Weird indenting here. With new stuff, generally running syntax_tree on the code is fine (DM me internally for details if needed)